torch/_dynamo/variables/builder.py

```diff
@@ -1702,7 +1702,6 @@ def wrap_tensor(self, value: torch.Tensor):
 
         grapharg = GraphArg(source, value, False, fake_tensor_value)
         tensor_proxy.node.meta["grapharg"] = grapharg
-        self.tx.output.add_symbol_bindings(grapharg)
```

Comment:
what's going on here?

Reply:
We've changed create_graph_input to always add symbol bindings [here](https://github.com/pytorch/pytorch/pull/138363/files#diff-1bbe80a42da83f0b0e1386c3aca92323930626a65173c7d7e838931446fb4324R2133), this is redundant.